src/view/screens/Debug.tsx

```diff
@@ -177,14 +177,17 @@ function ErrorView() {
 }
 
 function NotifsView() {
+  const {_} = useLingui()
   const triggerPush = () => {
     // TODO: implement local notification for testing
   }
   const triggerToast = () => {
-    Toast.show('The task has been completed')
+    Toast.show(_(msg`The task has been completed`))
```

Comment:
We can remove these are no point in localizing the `Debug.tsx` file

Reply:
It looks like there are other localized strings already in the `Debug.tsx` file, does that make any difference?